go.mod

```diff
@@ -4,13 +4,14 @@ go 1.12
 
 require (
 	github.com/agl/ed25519 v0.0.0-20170116200512-5312a6153412 // indirect
-	github.com/btcsuite/btcutil v0.0.0-20180706230648-ab6388e0c60a
+	github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d
 	github.com/cosmos/cosmos-sdk v0.37.0 // latest
-	github.com/cosmos/go-bip39 v0.0.0-20180618194314-52158e4697b8
+	github.com/cosmos/gaia v1.0.0
```

Comment:
This dependency is not necessary. Please revert any other unnecessary dependency changes that might have resulted from this.

Reply:
removed